.github/workflows/pylint.yml

```diff
@@ -0,0 +1,27 @@
+name: Pylint
```

Comment:
ok i will do this, but i will wait till this PR is approved because i have done a mistake and dont want balagan.
i have another 2 separtae PRs that i will submit after approvel. thanks!